routes/class.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var router = express.Router();
+const classes = require('../models/class')
+
+router.get('/', function (req, res, next) {
+    classes.find((err, clas) => {
+        if (err) {
+            console.error("couldnt get class", err)
+            res.send('couldnt get class');
+        } else {
+            res.json(clas)
+        }
+    })
+});
+
+router.post('/', function (req, res, next) {
+    console.log(req.body);
+    const languageTags = req.body.tags.split(',').map(val => val.trim());
```

Comment:
we could probably update the front end to send an array rather than a string in the post